docs/features/software-templates/writing-custom-actions.md

```diff
@@ -8,6 +8,40 @@ If you're wanting to extend the functionality of the Scaffolder, you can do so
 by writing custom actions which can be used along side our
 [built-in actions](./builtin-actions.md).
 
+---
+**NOTE**
+
+When adding custom actions, the actions array will **replace the built-in actions too**, so if you
+want to have those as well as your new one, you'll need to do the following:
+
+```ts
```

Comment:
It's a pretty large chunk of code to put at the top, what do you think about splitting it up into a heads up here and then the existing section below? A bit like the [docker multi-stage build warning](https://backstage.io/docs/deployment/docker#multi-stage-build)?

Reply:
It's just a move of existing content mainly because I spent 5 hours trying to work out why all actions disappeared and this pre-requisite happening _after_ the original instructions.

@Rugvip I only have a Markdown preview of the content, is there a way to render the pages so I can see what it looks like in production?

Reply:
Yep not saying this isn't an important change 😁  but I think we can get the same value a bit more stealthily

You can preview it by running the microsite, `cd microsite && yarn install && yarn start`

Reply:
I updated the Note block to be consistent with the rest of the site creating a slightly more subtle view of the prerequisite.

![image](https://user-images.githubusercontent.com/1300193/140988969-c4e5d33d-e070-4dcb-98b0-5d0e6f2ea97b.png)

Reply:
So I still think it's a bit much to have the entire codeblock up top in this section, feeling it would be enough with a notice at the top that reads something like "NOTE: Custom actions will replace the build-in ones, see the registration section below for how to keep using the build-in actions alongside custom ones"